src/v/config/configuration.cc

```diff
@@ -288,6 +288,12 @@ configuration::configuration()
       "Max duration before dispatching batched replicate requests",
       required::no,
       4ms)
+  , replicate_batching_window_max_size_bytes(
+      *this,
+      "replicate_batching_window_max_size_bytes",
```

Comment:
i think should be named `raft_replicate_batch_window_size`

Reply:
this should add notes in the release that we switched form 1MB to 128KB. 

cc: @dotnwat